src/main/java/de/hysky/skyblocker/skyblock/searchoverlay/SearchOverManager.java

```diff
@@ -106,36 +99,31 @@ private static void loadItems() {
             Object2ObjectMap<String, BazaarProduct> products = TooltipInfoType.BAZAAR.getData();
             for (Map.Entry<String, BazaarProduct> entry : products.entrySet()) {
                 BazaarProduct product = entry.getValue();
-            	String id = product.id();
-            	int sellVolume = product.sellVolume();
-            	if (sellVolume == 0)
-            		continue; //do not add items that do not sell e.g. they are not actual in the bazaar
-            	Matcher matcher = BAZAAR_ENCHANTMENT_PATTERN.matcher(id);
-            	if (matcher.matches()) {//format enchantments
-            		//remove ultimate if in name
-            		String name = matcher.group(1);
-            		if (!name.contains("WISE")) { //only way found to remove ultimate from everything but ultimate wise
-            			name = name.replace("ULTIMATE_", "");
-            		}
-            		name = name.replace("_", " ");
-            		name = capitalizeFully(name);
-            		int enchantLevel = Integer.parseInt(matcher.group(2));
-            		String level = "";
-            		if (enchantLevel > 0) {
-            			level = ROMAN_NUMERALS[enchantLevel - 1];
-            		}
-            		bazaarItems.add(name + " " + level);
-            		namesToId.put(name + " " + level, matcher.group(1) + ";" + matcher.group(2));
-            		continue;
-            	}
-            	//look up id for name
-            	NEUItem neuItem = NEURepoManager.NEU_REPO.getItems().getItemBySkyblockId(id);
-            	if (neuItem != null) {
-            		String name = Formatting.strip(neuItem.getDisplayName());
-            		bazaarItems.add(name);
-            		namesToId.put(name, id);
-            		continue;
-            	}
+                String id = product.id();
+                String name = product.name();
+                int sellVolume = product.sellVolume();
+                if (sellVolume == 0)
+                    continue; //do not add items that do not sell e.g. they are not actual in the bazaar
+                Matcher matcher = BAZAAR_ENCHANTMENT_PATTERN.matcher(name);
+                if (matcher.matches()) {//format enchantments
+                    name = matcher.group(1);
+                    if (!name.contains("Ultimate Wise")) {
+                        name = name.replace("Ultimate ", "");
+                    }
+
+                    name += " " + RomanNumerals.decimalToRoman(Integer.parseInt(matcher.group(2)));
+                    bazaarItems.add(name);
+                    namesToNeuId.put(name, id.substring(0, id.lastIndexOf('_')).replace("ENCHANTMENT_", "") + ";" + matcher.group(2));
```

Comment:
`matcher.group(2)` could be a variable called `level` or `enchantLevel` to make it easier to understand, like it was before the change.
```suggestion
                    int level = matcher.group(2);
                    name += " " + RomanNumerals.decimalToRoman(Integer.parseInt(level));
                    bazaarItems.add(name);
                    namesToNeuId.put(name, id.substring(0, id.lastIndexOf('_')).replace("ENCHANTMENT_", "") + ";" + level);
```

Reply:
It's a string not an int xd
sorry about that